IDE/WoCCode.md

```diff
@@ -0,0 +1,132 @@
+# Title
+
+WoCCode: Peer mentoring for the Women of Color coding community
+
+## Summary 
+
+Women of Color (WoC) -- which includes Black, indigenous, and Latinx
+minorities -- are the most underrepresented demographic from high
+paying industries, including software engineering. Yet WoC are flush
+with the talent, creativity, and social insight to solve complex
+problems and lead in the software industry. This program seeks to
+unite WoC and other members of minoritized groups, to battle feelings
+of isolation and marginalization that are prevalent in educational
+institutions and industry settings. Using a dedicated Slack space and
+cohorts of 4-5 people, we will create a peer mentoring network for WoC
+and members of other minoritized groups to engage, receive feedback,
+and troubleshoot issues around their participation in open source
+software communities, including Astropy.
+
+## Team 
+
+Program Lead: Lia Corrales (@eblur)
+
+Program Mentors and Slack moderators:
+* Lia Corrales
+* Kelle Cruz
+* TBD (invited Camille Avestruz)
+
+Anticipated 2020 cohort: 20 people from underrepresented groups in the
+United States (Black, indigenous, disabled, and gender-minority
+coders) or other underrepresented groups from the national stage (see
+Plan). We are open to any career stage, from undergraduate to research
+professionals.
+
+Guest webinar speakers: 6 industry or research professionals
+
+## Plan
+
+We will solicit applications to join the program via Twitter, AAS
+CSMA, and by emails to academic department chairs. We will accept
+applications from anyone, and will select a cohort in a way that
+prioritizes most marginalized demographics.
+
+We operate under the principle that supporting the most marginalized
+in our communities supports the well-being of all members of the
+Astropy community. While the Astropy project has shown considerable
+diversity in its international demographics, the majority of non-US
```

Comment:
I don't see any considerable diversity in the international demographics. The project is very heavily skewed towards USA based and beyond that towards native English speakers. Also, as far as I know there hasn't been any surveys that measured people's background such as class, childhood poverty, etc, especially not on the intentional setting. Naively I would avoid making any such assumptions into a half sentence such as this without data.
 
Maybe you want to remove that part as it doesn't add or take away from merit from the proposed project.

Reply:
No problem. I often hear our "list of international contributors" being bragged about when issues of diversity are discussed at Astropy meetings.  I am happy to take out the assumption of international demographics, since I don't know the numbers myself.  Regardless, I want to reiterate and make it clear that international participation alone is not the measure of an equitable and inclusive community.